src/patternfly/components/Nav/nav.scss

```
@@ -158,7 +187,13 @@
 }
 
 [class^="#{$nav}"][hidden] {
-  display: none;
+  visibility: hidden;
+
+  & .#{$nav}__list {
```

nit - the `&` is redundant here

```suggestion
  .#{$nav}__list {
```